internal/testutil/constant/pem_ca.go

```diff
@@ -0,0 +1,27 @@
+package constant
```

Comment:
I'd probably have DummyCert in internal/testutil/acc/stream_instance.go,   and refer it to it with acc.DummyCert

The problem I see with internal/testutil/constant is in case the test also needs internal/common/constant and it would need a package alias for one of them.   (maybe that doesn't happen frequently thought)

Reply:
left in external file close to stream_connection test and using embed feature